crypto/s2n_sequence.c

```diff
@@ -38,3 +42,17 @@ int s2n_increment_sequence_number(struct s2n_blob *sequence_number)
 
     return 0;
 }
+
+int s2n_sequence_number_to_uint64(struct s2n_blob *sequence_number, uint64_t *output)
+{
+    notnull_check(sequence_number);
+
+    uint8_t shift = 0;
+    *output = 0;
+
+    for (int i = sequence_number->size - 1; i >= 0; i--) {
+        *output += (uint64_t) sequence_number->data[i] << shift;
```

Comment:
Nitpick: I always find it hard to remember how tightly the cast bings.  I assume you mean to say 
```
*output += ((uint64_t) sequence_number->data[i]) << shift;
```
in which case the brackets might make it clearer.

Reply:
👍